bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/AbstractCULHandler.java

```diff
@@ -210,45 +206,21 @@ protected void notifyError(Exception e) {
      * @throws CULCommunicationException
      *             if
      */
-    protected void processNextLine() throws CULCommunicationException {
-        String deviceName = config.getDeviceAddress();
-        try {
-            String data = br.readLine();
-            if (data == null) {
```

Comment:
@johgoe I don't know what you mean here. processNextLine() was replaced by processNextLine(String), so any I/O error like EOF can't be handled in AbstractCULHandler any more.